pkgs/applications/misc/prusa-slicer/default.nix

```diff
@@ -71,29 +72,27 @@ let
 in
 stdenv.mkDerivation (finalAttrs: {
   pname = "prusa-slicer";
-  version = "2.9.0";
+  version = "2.9.1";
 
   src = fetchFromGitHub {
     owner = "prusa3d";
     repo = "PrusaSlicer";
-    hash = "sha256-6BrmTNIiu6oI/CbKPKoFQIh1aHEVfJPIkxomQou0xKk=";
+    hash = "sha256-i93P0W2/FQDpBSigKBFIi98fsx4ZAWh3g5fbqXzosss=";
     rev = "version_${finalAttrs.version}";
   };
 
-  # https://github.com/prusa3d/PrusaSlicer/pull/14010
-  patches = [(fetchpatch {
-    url = "https://github.com/prusa3d/PrusaSlicer/commit/cdc3db58f9002778a0ca74517865527f50ade4c3.patch";
-    hash = "sha256-zgpGg1jtdnCBaWjR6oUcHo5sGuZx5oEzpux3dpRdMAM=";
-  })];
-
-  # required for GCC 14
-  # (not applicable to super-slicer fork)
-  postPatch = lib.optionalString (finalAttrs.pname == "prusa-slicer") ''
-    substituteInPlace src/slic3r-arrange/include/arrange/DataStoreTraits.hpp \
-      --replace-fail \
-      "WritableDataStoreTraits<ArrItem>::template set" \
-      "WritableDataStoreTraits<ArrItem>::set"
-  '';
+  patches = [
+    (fetchpatch {
+      name = "build-with-boost-187";
+      url = "https://946495.bugs.gentoo.org/attachment.cgi?id=914594";
+      hash = "sha256-FVCpfkOe8a7zRG7QXoBu6i8I4sFu2i/j+E9uKPGmcec=";
```

Comment:
Somewhat unrelated: This hash seems to be incorrect.

Reply:
Just refetched, hash seems to be correct, unless gentoo bugtracker does something funny